.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# wasmcloud documentation reviewers
+* @autodidaddict @brooksmtownsend @LiamRandall @ricochet
```

Comment:
Figured we'd start using teams here's rather than individuals: https://github.com/orgs/wasmCloud/teams/docs-maintainers

Reply:
Whoops, yeah I should do that. Fixing

Reply:
Note I'll need to be added to that group :)

TY!